app-admin/pinky-bar/pinky-bar-9999.ebuild

```diff
@@ -0,0 +1,103 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+EGIT_REPO_URI="https://gitlab.com/void0/pinky-bar.git"
+
+inherit git-r3
+
+DESCRIPTION="Gather some system information and show it in this statusbar program"
+HOMEPAGE="https://gitlab.com/void0/pinky-bar"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="x11 alsa +net libnl +pci dvd sensors ncurses +colours weather mpd drivetemp smartemp"
+
+DEPEND="
+	sys-devel/m4
+	sys-apps/gawk
+	sys-devel/autoconf
+	>=sys-devel/automake-1.14.1
+"
+RDEPEND="
+	alsa? ( media-libs/alsa-lib )
+	x11? ( x11-libs/libX11 )
+	net? ( sys-apps/iproute2 )
+	libnl? ( >=dev-libs/libnl-3.2.27 dev-util/pkgconfig )
```

Comment:
pkgconfig belong in `DEPEND`, and needs to be from the `virtual` category, not `dev-util`